config/unix-g++.cmake

```diff
@@ -40,78 +26,41 @@ check_c_compiler_flag(   "-march=native" HAS_MARCH_NATIVE )
 
 #
 # Compiler Flags
-#
-# Consider using these diagnostic flags for Debug builds:
-# -Wcast-qual - warn about casts that remove qualifiers like const.
-# -Wstrict-overflow=4
-# -Wwrite-strings
-# -Wunreachable-code
-# - https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html
-#
+
 # Consider using these optimization flags:
 # -ffast-math -ftree-vectorize
 # -fno-finite-math-only -fno-associative-math -fsignaling-nans
 #
-# Added, but shouldn't be needed:
-# -Wno-expansion-to-defined - unable to use GCC diagnostic pragma to suppress warnings.
-#
 # Control FMA
 # -ffp-contract=off
 
 if( NOT CXX_FLAGS_INITIALIZED )
   set( CXX_FLAGS_INITIALIZED "yes" CACHE INTERNAL "using draco settings." )
-  string(APPEND CMAKE_C_FLAGS " -Wcast-align -Wpointer-arith -Wall -pedantic -Wfloat-equal"
-    " -Wunused-macros -fsanitize=bounds-strict -Wshadow -Wformat=2")
```

Comment:
Is the `fsanatize=bounds_strict` covered in one of these other flags?

Reply:
It is my understanding that the `sanitize=` flags must be requested individually.  The are not included in `-Wall` or the like.